templates/news-detail/news-detail.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable function-call-argument-newline, object-curly-newline, function-paren-newline */
+import { aside, div, a, button, strong, small } from '../../scripts/dom-helpers.js';
+import { createOptimizedPicture, getMetadata } from '../../scripts/aem.js';
+import { loadFragment } from '../../blocks/fragment/fragment.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main .section');
+  const $mainContent = div();
+  const articleChildren = [...$page.children];
+  [...articleChildren].forEach((child) => {
+    $mainContent.append(child);
+  });
+
+  // hero carousel
+  const $carousel = div({ class: 'hero-carousel wip' });
+  loadFragment('/news/news-detail/fragments/hero-carousel').then((frag) => {
+    $carousel.append(frag.firstElementChild.querySelector('.carousel-wrapper'));
+  });
+
+  // aside
+  const $aside = aside({ class: 'wip' });
+  loadFragment('/news/news-detail/fragments/right-sidebar').then((frag) => {
+    $aside.append(frag.firstElementChild.querySelector('.default-content-wrapper'));
+  });
+
+  // subhead
+  const $postMeta = small({ class: 'post-metadata' },
+    strong('Posted: '), getMetadata('publisheddate'),
+    ' | ',
+    strong('Categories: '), getMetadata('categories'),
+  );
+  const $h1 = $mainContent.querySelector('h1');
+  $h1.insertAdjacentElement('afterend', $postMeta);
+
+  // breadcrumbs
+  // TODO: make breadcrumbs a common function - perhaps a block
+  const $breadCrumbs = div({ class: 'breadcrumbs' },
+    a({ href: '/', 'arial-lable': 'View Home Page' }, 'Home'),
+    ' > ',
+    a({ href: '/news', 'arial-lable': 'View News Page' }, 'News'),
```

Comment:
Is this a typo?

Reply:
Why typo are you seeing? What am I missing?

Reply:
If you mean ' > ' - that is intentional, that's the breadcrumb separator.   I suppose it could be done in CSS with ::after - but the output markup from this is a little cleaner I think.